### ISSUE TITLE
"alter class C superclass A,B" fails. multiple inheritance superclass altering is undocumented

### ISSUE THREAD

**Issue by kstock** [orientechnologies/orientdb#4825]
This was encountered in 2.1.0

We wanted to alter a class to add OTriggered as a superclass.

http://orientdb.com/docs/last/SQL-Alter-Class.html
The documentation here doesn't have anything specific for multiple superclasses (I realize multiple inheritance is a new 2.1 feature, I haven't spotted yet where/if it is documented)

looking at the pull request that added this feature we saw:
https://github.com/orientechnologies/orientdb/pull/3802
```
create class C extends A, B;
alter class C superclasses A, B;
alter class C superclass +A;   --Add A class
alter class C superclass -B;   --Remove B class
```

The create, +A, -B stuff works.
however, when we tried to do
`alter class C superclasses A, B;`
we kept getting errors that class "A, B" was not found.

so this issue is for:

1)  the 
`alter class C superclasses A, B;`
syntax is not working

2) the various ways you can alter superclasses are not documented in the "ALTER CLASS" documentation

**Comment by nagarajasr** [comment]
works for me... in 2.1 GA

```
orientdb {db=nagu}> alter class myclass superclasses D, P

Class updated successfully

orientdb {db=nagu}>

```

**Comment by kstock** [comment]
Oh you are right, I feel so silly!

I could have sworn I tried using superclasses instead of superclass and it didn't work, but now it is working when I try it (I expect I was mistyping OTriggered!)

So I've removed:
"alter class C superclass A,B" fails. 
from the title because it is expected that that would fail because it should be superclasses.

The only issue then is that the ways to alter superclasses are still undocumented at:
http://orientdb.com/docs/last/SQL-Alter-Class.html
